refactor_persis_layer.html

```
@@ -0,0 +1,1027 @@
+    CreatedAt <span class="ty">time.Time</span>
+    UpdatedAt <span class="ty">time.Time</span>
+
+    <span class="cm">// ExpiresAt, when non-nil, hints that this record may be deleted after
+    // this time. Used for heartbeat / lease / TTL records.</span>
+    ExpiresAt *<span class="ty">time.Time</span>
+}
+
+<span class="cm">// ListQuery controls what List returns.</span>
+<span class="kw">type</span> <span class="ty">ListQuery</span> <span class="kw">struct</span> {
+    Prefix <span class="kw">string</span>     <span class="cm">// only records whose ID starts with this value</span>
+    Since  *<span class="ty">time.Time</span> <span class="cm">// CreatedAt >= Since</span>
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ > ].

(spec-char-escape)